collections-csharp/CsharpLists/TestLists/ListTests.cs

```diff
@@ -0,0 +1,92 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using CsharpLists;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace TestLists
+{
+    [TestClass]
+    public class ListTests
+    {
+        [TestMethod]
+        public void GivenEmptyList_AddItems()
+        {
+            var operations = new ListOperations();
+            var countries = operations.AddElements();
+            var otherCountries = new List<string>() { "Russia", "Bulgaria", "South Africa" };
+
+            Assert.AreEqual(countries.Count, otherCountries.Count);
+        }
+
+        [TestMethod]
+        public void GivenArray_AddItemsToList()
+        {
+            var operations = new ListOperations();
+            var countries = operations.AddElements();
+            var otherCountries = new string[3] { "Russia", "Bulgaria", "South Africa" };
+            countries.AddRange(otherCountries);
```

Comment:
Separate Arrange/Act/Assert with newlines in all methods please.

Reply:
Please have a look and let me know what you think. Thanks.